js/currencycom.js

```diff
@@ -91,10 +91,6 @@ module.exports = class currencycom extends Exchange {
                 'fetchTransfers': undefined,
                 'fetchWithdrawal': undefined,
                 'fetchWithdrawals': undefined,
-                'loadLeverageBrackets': undefined,
-                'loadMarkets': true,
-                'privateAPI': true,
-                'publicAPI': true,
```

Comment:
Do we remove `privateAPI/publicAPI` keys from derived classes from now and onwards?

Reply:
They should be filled-in upon building.